Library/Formula/pushpin.rb

```diff
@@ -0,0 +1,107 @@
+class Pushpin < Formula
+  homepage "http://pushpin.org"
+  url "http://packages.fanout.io/source/pushpin-1.0.0.tar.bz2"
+  sha1 "6862c69eda7eef2e7fc8dd15d9800a3b21792acd"
+
+  depends_on "pkg-config" => :build
+  depends_on "qt"
+  depends_on "zeromq"
+  depends_on "qca"
+  depends_on "qjson"
+  depends_on "mongrel2"
+  depends_on "zurl"
+
+  resource "pyzmq" do
+    url "https://pypi.python.org/packages/source/p/pyzmq/pyzmq-14.5.0.tar.gz"
+    sha1 "1dced02ea8527b5870ffdbe835d096aca5c01d2a"
+  end
+
+  resource "setproctitle" do
+    url "https://pypi.python.org/packages/source/s/setproctitle/setproctitle-1.1.8.tar.gz"
+    sha1 "a23463feac8d99b5504efc22f0ca2cfe2c145930"
+  end
+
+  resource "jinja2" do
+    url "https://pypi.python.org/packages/source/J/Jinja2/Jinja2-2.7.3.tar.gz"
+    sha1 "25ab3881f0c1adfcf79053b58de829c5ae65d3ac"
+  end
+
+  resource "tnetstring" do
+    url "https://pypi.python.org/packages/source/t/tnetstring/tnetstring-0.2.1.tar.gz"
+    sha1 "ffc35722f4ae978151acdc4801f20efa597c8b54"
+  end
+
+  def install
+    resource("pyzmq").stage { system "python", *Language::Python.setup_install_args(libexec/"vendor") }
+    resource("setproctitle").stage { system "python", *Language::Python.setup_install_args(libexec/"vendor") }
+    resource("jinja2").stage { system "python", *Language::Python.setup_install_args(libexec/"vendor") }
+    resource("tnetstring").stage { system "python", *Language::Python.setup_install_args(libexec/"vendor") }
+
+    inreplace "Makefile", /^prefix.*/, "prefix = #{prefix}"
```

Comment:
You can pass the variable to `make` as an argument to avoid needing to change them in the makefile:

```ruby
system "make", "prefix=#{prefix}", "varprefix=#{var}"
system "make", "install", "prefix=#{prefix}", "varprefix=#{var}"
```